### PR TITLE
HowTo PuTTY, WinSCP and add SSH keys

### PR DESCRIPTION
**Pull request by eleu**
cf. old Pull request https://github.com/camptocamp/c2cgeoportal/pull/1453

This is the new one based on 1.6

### CODE REVIEW

**Comment by asaunier** [comment]
It would be better to merge your commits in a single one. I can show you if needed.

**Comment by eleu** [comment]
ok done

File: doc/integrator/preparative_work.rst [hunk 1]
```diff
@@ -0,0 +1,74 @@
+.. _integrator_preparative_work:
+
+Creating a SSH Key with Windows
+===============================
+
+1. Download PuTTYgen:
+
+   a) Windows: http://the.earth.li/~sgtatham/putty/latest/x86/puttygen.exe
+   b) Other operating systems: http://www.chiark.greenend.org.uk/~sgtatham/putty/download.html
+
```
Comment (sbrunner) [inline doc/integrator/preparative_work.rst]: Steel have a trailing white space, the Travis will failed ...

**Comment by asaunier** [comment]
Looks good to me, I think you can merge when the tests pass.

**Pull request merged**